### ISSUE TITLE
cannot find ./pick

### ISSUE THREAD

**Issue by rbrcurtis** [marcuswestin/node-kafka#8]
fresh install of kafka, trying to require it throws the below.

```
Error: Cannot find module './pick'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (/Users/rcurtis/Code/bigbang/node_modules/kafka/node_modules/std/std.js:11:8)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
```

seems to work fine if you just comment out `pick: require('./pick')` in std.js

**Comment by nickdaugherty** [comment]
Same here, looks like a dependency is broken. OP should rename this issue to something descriptive :)

**Comment by rbrcurtis** [comment]
Done. :)  The issue is actually that there is a symlink in the repo, and symlinks aren't recognized by npm.  I thought this got fixed though..

**Comment by marcuswestin** [comment]
This should be done - I also just published the latest version v0.2.3 to npm.

Thanks guys!

Cheers,
Marcus
